### PR TITLE
Exclude Polyspace defects with status "Not a Defect" or "Justified"; Add substitution of env variables for min/max possible for Robot and Polyspace

### PR DESCRIPTION
**Pull request by JokeWaumans**
- Exclude Polyspace defects with a status "Not a Defect" or "Justified"
- Make it possible to use environment variables for `min` and `max` keys instead of numbers

### CODE REVIEW

**Comment by JasperCraeghs** [comment]
If you do not plan to fix your code in response to a result, assign one of the following statuses:

- Justified
- No Action Planned
- Not a Defect

https://nl.mathworks.com/help/polyspace_access/ug/fix-or-comment-polyspace-results-web-browser.html

Please document these changes.

**Review by Letme — APPROVED** [review]
Seems good to me.

**Comment by JokeWaumans** [comment]
> "If you do not plan to fix your code in response to a result, assign one of the following statuses:
> 
> * Justified
> * No Action Planned
> * Not a Defect"
> 
> https://nl.mathworks.com/help/polyspace_access/ug/fix-or-comment-polyspace-results-web-browser.html
> 
> Please document these changes.

Isn't "No Action Planned" a tricky one? 
"If you do not explicitly specify a status in your annotation, Polyspace assumes that you have set a status of No Action Planned."

**Comment by Letme** [comment]
I would as well avoid adding default status in here indeed. So current implementation seems fine.

File: README.rst [hunk 1]
```diff
@@ -324,6 +324,16 @@ For example, "run-time check" is the family of Code Prover and "defect" is the f
 The value of that key is a list, which contains the name of the column to check as a key and
 the value of that column to check together with ``min`` and ``max`` values.
 
+Note that all defects with one of the following statuses are excluded from the warnings and code quality report:
+- Justified
+- Not a Defect
```
Comment (JasperCraeghs) [inline README.rst:329]: This is rendered as a single paragraph https://github.com/melexis/warnings-plugin/tree/filter-poly-defects?tab=readme-ov-file#configuration

https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists

File: README.rst [hunk 2]
```diff
@@ -324,6 +324,18 @@ For example, "run-time check" is the family of Code Prover and "defect" is the f
 The value of that key is a list, which contains the name of the column to check as a key and
 the value of that column to check together with ``min`` and ``max`` values.
 
+Note that all defects with one of the following statuses are excluded from the warnings and code quality report:
+
+- Justified
+- Not a Defect
+
+These statuses are used when you do not plan to fix your code in response to a result.
+The status "No Action Planned" is not listed because this is a default status if you do not explicitly specfify
```
Comment (pietvr) [inline README.rst:333]:
```suggestion
The status "No Action Planned" is not listed because this is a default status if you do not explicitly specify
```

**Review by JasperCraeghs — CHANGES_REQUESTED** [review]

**Pull request merged**